include/grpc/grpc_security.h

```diff
@@ -240,6 +240,24 @@ GRPCAPI grpc_channel_credentials* grpc_ssl_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
     const verify_peer_options* verify_options, void* reserved);
 
+/** Sets the minimum TLS version that will be negotiated during the TLS
+   handshake. The caller MUST ensure that |creds| is an instance of
```

Comment:
nit: it is not a grpc tradition to wrap parameter with "|" in comment. you can just remove "|".